client.go

```diff
@@ -409,3 +410,24 @@ func (client *Client) GetBoundingBoxesVerbose() (out []BoundingBox, err error) {
 	}
 	return
 }
+
+// GetAvailableLanguages returns a list of available languages in the default tesspath
+func GetAvailableLanguages() ([]string, error) {
+	path := getDataPath()
```

Comment:
IMO,
If the change is only this, it's better to keep original source code position for `git blame` in future.

Reply:
I have reverted that part.